src/test/java/test/common/TestBase.java

```diff
@@ -12,28 +12,13 @@ public class TestBase {
 
     public String publicID = "2";
 
-    public String privateID = "12203162";
-
-    public int testId;
+    public static String testId;
```

Comment:
Ni ovo a i ni ovaj publicID nemoj da ti stoji u TestBase klasi, bolje nek se to nalazi u test klasi u kojoj ti treba